webcompat/form.py

```diff
@@ -65,6 +67,9 @@
 browser_test_label = u'Did you test in another browser?'
 textarea_label = u'Please describe what happened, including any steps you took before you saw the problem'  # noqa
 
+contact_message = u'There is a mistake in the username.'  # noqa
+contact_label = u'Anonymous is fine. Giving your Github nickname could help us with diagnosis. (publicly visible)'  # noqa
```

Comment:
nit: GitHub

Reply:
The wording probably needs some tweaking here... but I'm not sure what would be better. :)

Reply:
better wording: "WHO ARE YOU?"